fritm/hook.py

```diff
@@ -9,15 +9,15 @@
 SCRIPT = (Path(__file__).parent / "script.js").read_text()
 
 
-def spawn_and_hook(program, port=8080, filter="true"):
+def spawn_and_hook(program, port=8080, filter="true", verbose="false"):
```

Comment:
```suggestion
def spawn_and_hook(program, port=8080, filter="true", verbose=True):
```
Seems more logical to show logs bu default ?

Reply:
A verbose option is typically to add more output, so if the option is omitted, it should not be the default.